src/import/read.js

```diff
@@ -14,12 +14,22 @@ function parse(data, types) {
 
   types = (types==='auto') ? type.inferAll(data) : util.duplicate(types);
   cols = util.keys(types);
-  parsers = cols.map(function(c) { return type.parsers[types[c]]; });
+  parsers = cols.map(function(c) {
+    if (types[c]) {
+      var a = types[c].split(':', 2);
+      return {
+        parser: type.parsers[a[0]],
+        // optional arguments to be passed to the parser
+        opt: a.length > 1 ? a[1] : null
```

Comment:
In our template language, date-time format strings need to be enclosed in quotes. Should we be consistent with the pattern here?